include/seqan3/core/configuration/detail/all.hpp

```diff
@@ -0,0 +1,74 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2020, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2020, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+/*!\file
```

Comment:
Isn't it a problem that we have all this nice documentation here but as it is in a detail header it is invisible to users?

Reply:
Yes, you're right. I move this to the configutation/all.hpp